timemachine/md/hrex.py

```diff
@@ -213,13 +206,16 @@ def attempt_neighbor_swaps_fast(
         n_swap_attempts : int
             number of individual swap attempts, each between a randomly-selected neighbor pair
 
+        seed : int
```

Comment:
super nit:
```suggestion
        seed: int
```

Reply:
It seems we have a mixed convention. Fwiw, the numpydoc style guide includes the extra space: https://numpydoc.readthedocs.io/en/latest/format.html

Maybe I'll update so we're at least consistent within a given docstring for now?

Reply:
I applied the other suggested fix. Will leave this one as-is for now for consistency.